lib/peek/rblineprof/controller_helpers.rb

```diff
@@ -120,7 +112,7 @@ def inject_rblineprof
               end
             end
             output << "<pre class='duration'>#{times.join("\n")}</pre>"
-            output << "<div class='code'>#{pygmentize(file_name, code.join, 'ruby')}</div>"
+            output << "<div class='code'>#{highlight(file_name, code.join, true)}</div>"
```

Comment:
I think it would be great to use a `<pre>` instead of a `<div>` here, or at least allow to customize that.